src/core/tests/Test_StatusHandler.py

```diff
@@ -349,6 +349,30 @@ def test_status_file_maintenance_run_id(self):
         self.runtime.status_handler.set_installation_substatus_json()
         self.assertEqual(expected_maintenance_run_id, self.get_status_handler_substatus_maintenance_run_id())
 
+    def test_sequence_number_changed_termination_auto_assess_only(self):
+        self.runtime.execution_config.exec_auto_assess_only = True
+        self.runtime.status_handler.report_sequence_number_changed_termination()
+        with self.runtime.env_layer.file_system.open(self.runtime.execution_config.status_file_path, 'r') as file_handle:
+            substatus_file_data = json.load(file_handle)[0]["status"]["substatus"][0]
+            self.assertTrue(substatus_file_data["name"] == Constants.PATCH_ASSESSMENT_SUMMARY)
+            formatted_message = json.loads(substatus_file_data['formattedMessage']['message'])
+            self.assertTrue(formatted_message["errors"]["details"][0]["code"] == Constants.PatchOperationErrorCodes.NEWER_OPERATION_SUPERSEDED)
```

Comment:
Add the validation:
self.assertEqual(json.loads(substatus_file_data["formattedMessage"]["message"])["startedBy"], Constants.PatchAssessmentSummaryStartedBy.PLATFORM)

All Auto Assessment operations should have startedBy set to Platform. This will validate the execution follows auto assessment path correctly